packages/atst/src/cli.ts

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
+import { cac } from 'cac'
+import type { Address } from '@wagmi/core'
+
+import { readOptionsValidators, ReadOptions } from './commands/read'
+import * as logger from './lib/logger'
+// @ts-ignore it's mad about me importing something not in tsconfig.includes
```

Comment:
Yea I actually could add it to tsconfig here with no issues.   Normally it's an issue because as soon as you add something not in `src` you ruin the folder structure of dist like this

before: `dist/index.js`
after: `dist/src/index.js`

Usually I prefer to keep the folder structure clean.  Here we are building with tsup though not tsc so it's not relavent.  I will add it to tsconfig before landing.